aiida_project/project/venv.py

```diff
@@ -25,12 +37,21 @@ class VenvProject(BaseProject):
 
     def create(self, python_path: Path) -> None:
         super().create(python_path)
-        venv_command = [f"{python_path.resolve()}", "-m", "venv", "."]
         self.venv_path.mkdir(
             exist_ok=True,
             parents=True,
         )
-        subprocess.run(venv_command, cwd=self.venv_path, capture_output=True)
+        venv_command = [
+            UV_EXE,
+            "venv",
+            "--no-project",
+            "--allow-existing",
+            "-p",
+            f"{python_path.resolve()}",
+            str(self.venv_path),
+        ]
+        subprocess.run(venv_command, check=True, capture_output=True)
+        self.install(["pip"])
```

Comment:
Alright, I still discovered one issue while testing: When creating the environment with `uv`, it doesn't automatically install `pip` in it. So I added this line here. Any objections @danielhollas?